en/docs/apis/api-concepts.md

```diff
@@ -1,23 +1,70 @@
 # API Concepts
-Before you start managing APIs via Choreo, there are basic concepts that you need to understand. This section walks you through all the concepts you need to know. 
+Choreo's API management capabilities allow you to create APIs, manage APIs, and expose services and external APIs as managed APIs. Here are some basic concepts which will help you understand API management in Choreo as you create and manage your APIs in the [Choreo Console]({{choreo_console}}).
  
 ## API
-An API in Choreo allows you to expose business functionality as a service to customers, partners, suppliers, and other stakeholders.
+An API is an intermediate layer that acts as a communication protocol between a consumer and a service, simplifying the consumption of the service. In addition to hiding the underlying implementation details of a service, an API provides a secure, controlled, and well-documented approach to access an exposed service.
+
+## API Format
+Open API Specification (OAS, a.k.a Swagger) format is the underlying representation of an API in Choreo. You can import an API using a Swagger definition in the Publisher Portal. Additionally, a user can view, edit, import, or download an API definition in OAS format via the API Publisher Portal.
```

Comment:
Should we say OpenAPI instead of Open API?

Reply:
Yes. Fixed.